Sources/UtiliKit/General/URL+Extensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  URL+Extensions.swift
+//  UtiliKit-iOS
+//
+//  Created by William McGinty on 7/22/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+
+extension URL: ExpressibleByStringLiteral {
+    
+    /// Creates a `URL` from a string literal object. This initializer will `fatalError` if given an invalid URL string.
```

Comment:
Also might want to add some additional info on how this is intended to be used (only for static "pre-baked" strings and not dynamically generated ones).
```
/// Creates a `URL` from a string literal object. This initializer will `fatalError` if given an invalid URL string. For this reason, you should only use this for URLs that are baked-in to your app at compile time. We do *not* recommend using this to create URLs from strings downloaded at run time.
```

Reply:
I believe the compiler will enforce that will it not? It has to be a string literal so anything dynamic won't pass the type check.

Reply:
I'm not sure exactly when the compiler will enforce things. This seems to work okay, but I would argue that it's not how this extension should be used:
```swift
let page = 0
let myURL = "http://apple.com?page=\(page)"
```

Reply:
Fair enough, will add a 'use for static pre-defined URLs only' comment.

Reply:
Although maybe that's a poor example since `page` is known at compile-time...